src/components/profile/profile.js

```diff
@@ -120,6 +121,40 @@ export class Profile extends React.Component {
     if (data)
       return (
         <div style={{ position: "sticky", top: 0 }}>
+          {preview && (
+            <Helmet>
+              <title>{person_data.fullName}</title>
+
+              {/* Google  */}
+              <meta name='author' content={person_data.fullName} />
+              <meta name='description' content={data.desc} />
```

Comment:
`description` should be optional as well. Maybe you can come up with a default description?